src/main.js

```diff
@@ -114,8 +121,11 @@ const bot = new Client({
 
 bot.on("ready", async () => {
   // when loaded (ready event)
-  bot.user.setActivity(`${PREFIX}help | ${PREFIX}info`, { type: "PLAYING" });
+  bot.user.setActivity(configManager.config.activity.description ?? "nil", {
+    type: configManager.config.activity.type.toUpperCase() ?? "PLAYING",
```

Comment:
Need to move the `??` so it's in parens with `configManager.config.activity.type`. When activity key is empty but present:
```
[2022-04-24 18:29:13] [error] uncaughtException: Cannot read properties of undefined (reading 'toUpperCase')
TypeError: Cannot read properties of undefined (reading 'toUpperCase')
    at Client.<anonymous> (/home/brandon/Documents/git/devcodeabode/MoDiBo/src/main.js:125:46)
    at Client.emit (node:events:520:28)
    at WebSocketManager.triggerClientReady (/home/brandon/Documents/git/devcodeabode/MoDiBo/node_modules/discord.js/src/client/websocket/WebSocketManager.js:384:17)
    at WebSocketManager.checkShardsReady (/home/brandon/Documents/git/devcodeabode/MoDiBo/node_modules/discord.js/src/client/websocket/WebSocketManager.js:367:10)
    at WebSocketShard.<anonymous> (/home/brandon/Documents/git/devcodeabode/MoDiBo/node_modules/discord.js/src/client/websocket/WebSocketManager.js:189:14)
    at WebSocketShard.emit (node:events:520:28)
    at WebSocketShard.checkReady (/home/brandon/Documents/git/devcodeabode/MoDiBo/node_modules/discord.js/src/client/websocket/WebSocketShard.js:475:12)
    at WebSocketShard.onPacket (/home/brandon/Documents/git/devcodeabode/MoDiBo/node_modules/discord.js/src/client/websocket/WebSocketShard.js:447:16)
    at WebSocketShard.onMessage (/home/brandon/Documents/git/devcodeabode/MoDiBo/node_modules/discord.js/src/client/websocket/WebSocketShard.js:301:10)
    at WebSocket.onMessage (/home/brandon/Documents/git/devcodeabode/MoDiBo/node_modules/ws/lib/event-target.js:199:18)
    ```